.github/ISSUE_TEMPLATE/feature-request.yaml

```diff
@@ -0,0 +1,90 @@
+name: Feature request or enhancement 💡
+description: Suggest a new component/pattern/idea for this project.
+title: '[Feature Request]: '
+labels: 'type: enhancement 💡'
+body:
+  - type: markdown
+    attributes:
+      value: '## Welcome!'
+  - type: markdown
+    attributes:
+      value:
+        Use this template if you want to request a new feature, or a change to
+        an existing feature. If you are reporting a bug or problem, please use
+        the bug template instead.
+  - type: textarea
+    id: problem
+    attributes:
+      label: The problem
+      description:
+        'Provide a clear and concise description of what the problem this new
+        feature is trying to solve. (For example, I find it frustrating
+        when...). Be sure to attach screenshots or videos to illustrate the
+        problem.'
+    validations:
+      required: true
+  - type: textarea
+    id: solution
+    attributes:
+      label: The solution
+      description:
+        'Provide a clear and concise description of what you would like to
+        happen instead. Be sure to attach screenshots or videos to illustrate
+        the solution if possible. Design specs are highly encouraged.'
+    validations:
+      required: true
+  - type: textarea
+    id: design
+    attributes:
+      label: Design link
+      description: Provide link to design of example of existing component
+    validations:
+      required: true
```

Comment:
Hmm, thoughts on whether we can make this one optional? We sometimes get enhancement requests specifically from development (truncation, string to node), etc. But maybe having a design that supports it still makes sense. 🤔

Reply:
yeah good point, I'll remove the requirement in the case for dev feature requests